src/plumbing/main.rs

```diff
@@ -1221,13 +1221,17 @@ pub fn main() -> Result<()> {
         },
         Subcommands::Completions { shell, out_dir } => {
             let mut app = Args::command();
+
+            let bin_name = "gix";
+            app.set_bin_name(bin_name);
```

Comment:
This is not necessary, but could be used elsewhere if we wanted to `app.get_bin_name()`.